cpp/src/parquet/encoding.cc

```diff
@@ -2163,6 +2164,8 @@ class DeltaBitPackEncoder : public EncoderImpl, virtual public TypedEncoder<DTyp
   uint32_t total_value_count_{0};
   UT first_value_{0};
   UT current_value_{0};
+  UT min_delta_;
```

Comment:
should it default as UT::max ?

Reply:
This could be done but would not change behavior as min_delta__ is set to the first value when encoding starts.

Reply:
It will now be initialized to its maximum value.
Even though this gets overwritten at the start of encoding your suggestion is a good idea to make sure nothing breaks in case encoding initialization changes in the future.